cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/helpers/Util.kt

```diff
@@ -470,6 +472,23 @@ object Util {
         conditionNodes.forEach { n.prevDFGEdges += it }
     }
 
+    /**
+     * Returns the RootPath that belongs to the absolute file path in `file` amongst the
+     * `rootPaths`.
+     *
+     * @param file absolute file to find the rootPath for
+     * @param rootPaths candidates of root paths for the file
+     * @return the first identified root path candidate
+     */
+    fun getRootPath(file: File, rootPaths: List<Path>): Path {
```

Comment:
`fun File.rootPath(rootPaths: List<Path>)` would be more kotlin-like. Also there is no real need to put it into the `Util` object, we can just have it as a top-level func.